main.go

```diff
@@ -64,40 +52,23 @@ func main() {
 		runSetup(yk)
 	} else {
 		if *socketPath == "" {
-			flag.Usage()
-			os.Exit(1)
+			if runtime.GOOS == "windows" {
+				*socketPath = "\\\\.\\\\pipe\\\\openssh-ssh-agent"
```

Comment:
I would avoid hardcoding the path where the pipe is implemented. If someone wants to customize it, there should exist a way to do it. Similar to Mac and Linux.

Reply:
I think you might be misreading this line -- this is meant to set a default on Windows only (the flag is required on Unix systems). Maybe it would be better to set a default up above in the `flag` section instead.

Reply:
You’re right I screwed this up. Sorry for the hassle.